test/e2e/framework/poddisruptionbudget.go

```diff
@@ -0,0 +1,28 @@
+package framework
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	policyv1 "k8s.io/api/policy/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+// CreatePodDisruptionBudget create PodDisruptionBudget.
```

Comment:
create -> creates

Reply:
FWIW, this is following convention:

[ConfigMap](https://github.com/karmada-io/karmada/blob/9b8a2b5a184d459af5e189ca12176e994da9297d/test/e2e/framework/configmap.go#L17)
[Ingress](https://github.com/karmada-io/karmada/blob/9b8a2b5a184d459af5e189ca12176e994da9297d/test/e2e/framework/ingress.go#L14)
[Daemonset](https://github.com/karmada-io/karmada/blob/9b8a2b5a184d459af5e189ca12176e994da9297d/test/e2e/framework/daemonSet.go#L14)
[Job](https://github.com/karmada-io/karmada/blob/9b8a2b5a184d459af5e189ca12176e994da9297d/test/e2e/framework/job.go#L17)